categories.R

```diff
@@ -0,0 +1,119 @@
+setwd("/Users/floriankaiser/Downloads/wt21-nearby")
+
+library(tidyverse)
+
+df <- read_csv2("data_preprocessed.csv")
+
+# Work & Education:
+education <- 
+  c(
+  "Arbeit",
+  "Architektur",
+  "Beratung",
+  "Bibliothek",
+  "Bildung",
+  "Gesundheit",
+  "Kommunikation",
+  "Kultur",
+  "Kunst",
+  "Literatur & Lesungen",
+  "Musik",
+  "VHS"
```

Comment:
Instead of repeating raw strings in different places, we should extract them as constants, defining in only one place, reusing everywhere else in the code.
This way when we have to do any change to the code, every part of the code base using the constant that is getting changed will automatically use the right value.